manual/mapper/entities/README.md

```diff
@@ -159,6 +159,67 @@ private int day;
 This information is used by some of the DAO method annotations; for example,
 [@Select](../daos/select/)'s default behavior is to generate a selection by primary key.
 
+#### Computed properties
+
+Annotating an entity property with [@Computed] indicates that when retrieving data with the mapper
+this property should be set to the result of a computation on the Cassandra side, typically a
+function call:
+
+```java
+private int v;
+
+@Computed("writetime(v)")
+private long writetime;
+```
+
+The CQL return type of the formula must match the type of the property, otherwise an exception
+will be thrown.
+
+[@Computed] does not support case-sensitivity. If the expression contains case-sensitive column
+or function names, you'll have to escape them:
+
+```java
+@Computed("\"myFunction\"(\"myColumn\")")
+private int f;
+```
+
+[@Computed] fields are only used for queries, so they will not be considered for [@Update] or
```

Comment:
To clarify I would add: `[@Computed] fields are only used for [@Select] queries, so they will not be considered for [@Update]...`

Reply:
It also works for `@GetEntity` and `@Query` as well (see sections below), so I think it makes sense as written, but can adjust it if you prefer.

Reply:
It's just that, at least to my ears, "queries" can refer to any form of CQL query, including inserts and updates. But OK to leave it as is if you think it's clear enough.

Reply:
Maybe we could list them explicitly.

Reply:
updated to `select-based` queries so it's more clear.